java/code_to_gast/java_assign.py

```diff
@@ -1,20 +1,24 @@
 import java.code_to_gast.java_router as java_router
-
 """
 Handles java var declarations to generic AST node
 """
+
+
 def assign_to_gast(node):
     gast = {"type": "varAssign", "kind": "let"}
     gast["varValue"] = java_router.node_to_gast(node.initializer)
     # var name stored as string but we don't want to return gast string node
     gast["varId"] = {"type": "name", "value": node.name}
     return gast
 
+
 """
 Handles increment and decrement operators
 """
+
+
```

Comment:
I don't think we want space between our doc string and function

Reply:
+1. Is there a way to configure this?

Reply:
Python doc strings usually go _inside_ the function, I think that might be why it's getting confused.